examples/self-signed-cert/go/go.mod

```diff
@@ -3,56 +3,58 @@ module github.com/pulumi/pulumi-tls/examples/self-signed-cert/go
 go 1.19
 
 require (
-	github.com/pulumi/pulumi-tls/sdk/v4 v4.5.0
-	github.com/pulumi/pulumi/sdk/v3 v3.50.2
+	github.com/pulumi/pulumi-tls/sdk/v5 v5.0.0
+	github.com/pulumi/pulumi/sdk/v3 v3.56.0
 )
 
+replace github.com/pulumi/pulumi-tls/sdk/v5 => ../../../sdk
```

Comment:
Having this replace in makes it compilable before a 5.x release, though I think ProgramTest makes it possible to fix it up in flight.